main.js

```diff
@@ -0,0 +1,113 @@
+const menuBtn = document.querySelector('.menu-btn');
+const navbar = document.getElementById('navbar');
+const navbarMenu = document.querySelector('.navbar-right');
+const featureList = document.querySelector('.feature-list');
+const showMore = document.querySelector('.more-btn');
+
+// Feature data Object
+const featureData = [
+  {
+    name: 'Baraka Danny',
+    role: 'Full Stack Developer',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-1.jpg',
+  },
+  {
+    name: 'Sammy Lee',
+    role: 'creative director',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-2.jpg',
+  },
+  {
+    name: 'John Doe',
+    role: 'Team leader/ CEO',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-3.jpg',
+  },
+  {
+    name: 'Jane Hill',
+    role: 'Network Engineer/ Developer',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-4.jpg',
+  },
+  {
+    name: 'zoey mary',
+    role: 'Accountant Admin',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-5.jpg',
+  },
+  {
+    name: 'Tina Smith',
+    role: 'Front End Developer',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod, nisl eget consectetur sagittis.',
+    image: 'img/profile-3.jpg',
+  },
+];
+
+// burger menu action
+let menuOpen = false;
+
+menuBtn.addEventListener('click', () => {
+  if (!menuOpen) {
+    menuBtn.classList.add('open');
+    menuOpen = true;
+  } else {
+    menuBtn.classList.remove('open');
+    menuOpen = false;
+  }
+});
+
+// sticky navigation menu
+window.onscroll = function () {
+  if (window.pageYOffset > 100) {
+    // header.classList.add('sticky');
+    navbar.classList.add('sticky');
+  } else {
+    // header.classList.remove('sticky');
```

Comment:
- [ ] It will be better to remove unused code to make your design clean and your code be consistent with JavaScript best practices.